docs/task_information.rst

```diff
@@ -170,3 +170,19 @@ the output could be like:
 Delete Unnecessary Output Files
 --------------------------------
 To delete output files which are not necessary to run a task, add option ``--delete-unnecessary-output-files``. This option is supported only when a task outputs files in local storage not S3 for now.
+
+
+
+Dump TaskInfo Table
+--------------------
+When using ``gokart.build()``, you can dump information of dependent tasks to a file as TaskInfo table.
```

Comment:
[ask] Is it difficult to dump the information on standard out or error as a log, too?

Reply:
@hirosassa 
If you want to dump the information as a standard out, I think of using `make_tree_info` may be better.